www/~/%username/statement.json.spt

```diff
@@ -15,6 +15,10 @@ if request.method == 'POST':
     if lang not in LANGUAGES_2:
         raise Response(400, "unknown lang")
 
+    # Statements are limited to 256kB to avoid DoS'ing the markdown parser.
+    if len(content) > (256 * 1024):
+        raise Response(400, "Statement too big!")
```

Comment:
Since this can show up in the UI, we should i18n it (the "unknown lang" just before should only be hittable from a script).

![screen shot 2016-08-22 at 11 26 05 pm](https://cloud.githubusercontent.com/assets/134455/17879440/fdcd593e-68bf-11e6-80d2-e55ac9736fdd.png)